### PR TITLE
Patient csv upload fixes

### PR DESCRIPTION
**Pull request by TomNUSDS**
## Related Issue or Background Info

- Trying to use patient upload to upload 1000+ fake patients and hit some issues.
- Used mockaroo to create a schema and generate upload files: https://www.mockaroo.com/a228f3d0

## Changes Proposed

- Detect and handle uploads with and without an initial column row 
- Update/fix unit tests

### CODE REVIEW

File: backend/src/main/java/gov/cdc/usds/simplereport/service/UploadService.java [hunk 1]
```diff
@@ -78,7 +90,7 @@ public String processPersonCSV(InputStream csvStream) throws IllegalGraphqlArgum
             rowNumber++;
             try {
                 _ps.addPatient(
-                    parseUUID(row.get(FACILITY_ID)),
+                    parseUUID(row.get(FACILITY_ID)),    // todo: verify this is a facility in the current Org.
```
Comment (TomNUSDS) [inline backend/src/main/java/gov/cdc/usds/simplereport/service/UploadService.java:93]: This is potentially a big mistake if the spreadsheet has the wrong `facilityId`.

Comment (timothybest-usds) [reply backend/src/main/java/gov/cdc/usds/simplereport/service/UploadService.java:93]: This is already handled by `updatePersonFacility` which is called from `addPatient`

**Review by timothybest-usds — APPROVED** [review]
The Mackaroo link seems awesome. Can you add it to the README or wiki. It would also be nice to load a 1000 patient file up to the dev env

File: backend/src/main/java/gov/cdc/usds/simplereport/service/UploadService.java (hunk 1 above)
Comment (TomNUSDS) [reply backend/src/main/java/gov/cdc/usds/simplereport/service/UploadService.java:93]: What I mean by this is that the facilityId guid comes from the spreadsheet.

**Pull request merged**